kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1075,11 +1075,27 @@ public void onChannelUpdated() {
             return;
         }
 
-        channel = ChannelService.getInstance(getActivity()).getChannelByChannelKey(channel.getKey());
+        String existingAssignee = channel.getConversationAssignee();
 
+        channel = ChannelService.getInstance(getActivity()).getChannelByChannelKey(channel.getKey());
         //conversation is open
         //if the conversation is opened from the dashboard while the feedback input fragment is open, the feedback fragment will be closed
         setFeedbackDisplay(channel != null && channel.getKmStatus() == Channel.CLOSED_CONVERSATIONS && !KmUtils.isAgent(getContext()));
+
+        if (existingAssignee != null && !existingAssignee.equals(channel.getConversationAssignee())) {
```

Comment:
use !channel.getConversationAssignee().equals(...) then null check will not be required

Reply:
channel.getConversationAssignee() might also be null, that's why placed it inside so that we can avoid that null check.